tests/unit/lib/build_module/test_workflow_config.py

```diff
@@ -51,6 +51,17 @@ def test_must_work_for_provided(self, runtime):
         self.assertEqual(len(EventTracker.get_tracked_events()), 1)
         self.assertIn(Event("BuildWorkflowUsed", "provided-None"), EventTracker.get_tracked_events())
 
+    @parameterized.expand([("provided",)])
```

Comment:
Should this be changed to provided.al2 as well? 🤔

Reply:
Yeap good catch, updated unit tests with that